service/lotusclient/lotusclient.go

```diff
@@ -99,7 +99,7 @@ func (c *Client) ImportData(pcid cid.Cid, file string) error {
 	}
 	ctx, cancel := context.WithTimeout(c.ctx, requestTimeout)
 	defer cancel()
-	if err := c.c.DealsImportData(ctx, pcid, file); err != nil {
```

Comment:
The code of the two are exactly the same

https://github.com/filecoin-project/lotus/blob/master/node/impl/storminer.go#L627-L635

https://github.com/filecoin-project/lotus/blob/master/node/impl/storminer.go#L416-L424